### ISSUE TITLE
[match details] Start a match which launch a scoreboard

### ISSUE THREAD

**Issue by eric-taix** [eric-taix/volley34-mobile#148]

**Comment by eric-taix** [comment]
I was working on this issue and I realized that this page may have a lot of actions:
1- Add the match to the phone calendar (currently implemented)
2- Route to the gymnasium (currently implemented)
3- Download the match sheet (see #147)
4- Start a scoreboard  (see #148)
5- Send the result just by filling the result in a form (related to #189)
6- Postpone a match (not referenced but similar to #189)

Currently the two first actions are implemented using buttons which is fine for 2 actions but with 6 actions (in the future), it will not be ergonomic. There are probably plenty of way to implement this (and ideas are welcome), here's the first one I thought: a floating circular menu which opens actions. I didn't use toolbar actions (top right corner) as it's not very explicit for the user (usually used for advanced actions). WDYT?

cc @matt-aubert 

![match-result](https://user-images.githubusercontent.com/712517/146361203-4a89bebf-e6e1-41bf-aee7-4681efe87811.gif)

**Comment by eric-taix** [comment]
I was not very happy with the first version. I tried to only use the circular menu for the match actions which seems better and use TextButton (secondary actions) for the direction and the agenda
![Screenshot_1639654747](https://user-images.githubusercontent.com/712517/146365337-79ee89aa-0d08-4057-99a8-dae388c9a33b.png)

**Comment by matt-aubert** [comment]
Nice !
I agree to keep some actions in buttons.

Perhaps moving the two buttons at the left side of the screen to avoid being covered by the circular menu ?

And ideally : the first access to this screen triggers a tour of the functionalities

**Comment by eric-taix** [comment]
👍 Great idea to align text buttons on the left

**Comment by eric-taix** [comment]
Here is the new page design:
- The page title is just "Match" or "Event" or "Tournament"
- For a match, the first bloc displayed is the opponents
- We can now click on a opponent to open the team details
- Agenda and itinerary buttons are `TextButton`
- Other actions are in the circular buttons
- Actions not yet implemented will be displayed but disabled otherwise with 2 options, the menu is awful
- During the first access, a feature tour of the page will be displayed
![Screenshot_1639667249](https://user-images.githubusercontent.com/712517/146396870-4dca35cf-a254-49da-901f-dfd561c992ef.png)

**Comment by matt-aubert** [comment]
Great work ! looking forward to check that out

**Comment by Lo0ping** [comment]
be carrefull !!! 
scoreboard must not replace official match scoresheet or you do a complete match scoresheet including players listing and "signing" process ...

**Comment by eric-taix** [comment]
Yes you're right. For now the goal of this feature is just to provide a tool but will not replace the official sheet until (as you said) we have login, signing capabilities and licence entered.

**Comment by clapoujade** [comment]
About the score board.
Never forget that a feature is good if it eases user's lives.

My asumption is only valid if we consider the real paper score board will remain the only reference to rely on 

Having another score board will bring two new issues:
- user will have to write the results a second time in the 'legacy' paper score board, in order to send it.
- if one user is using this alternative AND another one is using the legacy paper score board....having two different source for the score result could cause real issue.
So, based on my asumption, what is the added value for this new feature

**Comment by eric-taix** [comment]
Your assumption is correct today (see the requirements in previous comments about signing capabilities, licence numbers, ...). But this assumption may change in the future. 

However, many teams your physical scoreboard during the match. Then at the end of each set, they report the set's score on the score sheet. So this tool is just a replacement of a physical score board and will automatically fill the match's result form, not more not less for now.

I'll not enter into the features we can develop in the future. Let's keep that discussion for another time.
BTW: I've enabled the "Discussion" tab in Github. I think this is the right place to have this kind of discussion (the future of this app and volley34)
